prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Todo {
+  id        String    @id @default(uuid())
+  title     String    @unique
+  completed Boolean   @default(false)
+  createdAt DateTime  @default(now())
+  updatedAt DateTime? @updatedAt
+}
+
+model User {
+  id        String    @id @default(uuid())
+  email     String?   @unique
+  password  String?
+  createdAt DateTime? @default(now())
+  updatedAt DateTime?
```

Comment:
You can use the method `@updatedAt` in Prisma for automation update the time when we update a record
```ts
  updatedAt DateTime  @updatedAt
```